src/python/BFinitPassthrough.py

```diff
@@ -132,7 +133,10 @@ def reset_to_bootloader(args):
     if rx_target == "":
         dbg_print("Cannot detect RX target, blindly flashing!")
     elif rx_target != flash_target:
-        raise WrongTargetSelected("Wrong target selected your RX is '%s', trying to flash '%s'" % (rx_target, flash_target))
+        if query_yes_no("\n\n\nWrong target selected! your RX is '%s', trying to flash '%s', continue? Y/N\n" % (rx_target, flash_target)):
```

Comment:
Do we want to add some colour formating so it pops out to users?

https://github.com/ExpressLRS/ExpressLRS/blob/4b6a56d53e1c123f30dd52d028e99814b22c74b8/src/python/build_flags.py#L16

Reply:
That never seemed to work in my console